editor/src/core/shared/optics/optic-creators.ts

```diff
@@ -79,6 +79,28 @@ export function fromArrayIndex<A>(index: number): Optic<Array<A>, A> {
   )
 }
 
+export function pairTupleFirst<A1, A2>(): Optic<[A1, A2], A1> {
+  return lens(
+    ([a1, _a2]) => {
+      return a1
+    },
+    ([_a1, a2], newA1) => {
+      return [newA1, a2]
+    },
+  )
+}
+
+export function pairTupleSecond<A1, A2>(): Optic<[A1, A2], A2> {
+  return lens(
+    ([_a1, a2]) => {
+      return a2
+    },
+    ([a1, _a2], newA2) => {
+      return [a1, newA2]
+    },
+  )
+}
```

Comment:
What are these for?

Reply:
They were implemented at one point to get at the respective parts of the result that comes back from `Object.entries`, but they ended up not being used in the end.